pinot-core/src/test/java/org/apache/pinot/core/util/SchemaUtilsTest.java

```diff
@@ -136,13 +138,17 @@ public void testCompatibilityWithTableConfig() {
     schema = new Schema.SchemaBuilder().setSchemaName(TABLE_NAME)
         .addDateTime(TIME_COLUMN, DataType.LONG, "1:MILLISECONDS:EPOCH", "1:HOURS").build();
     tableConfig =
-        new TableConfigBuilder(TableType.REALTIME).setTableName(TABLE_NAME).setTimeColumnName(TIME_COLUMN).build();
+        new TableConfigBuilder(TableType.REALTIME).setTableName(TABLE_NAME)
+            .setStreamConfigs(Map.of(StreamConfigProperties.STREAM_TYPE, "foo"))
```

Comment:
Is this correct?  Do we need topic name, consumer type etc? Same for other tests

Reply:
> Is this correct? Do we need topic name, consumer type etc? Same for other tests

Yeah we need to improve validations for our realtime table configs. It's the same for all other tests.

Reply:
I added a validation in TableConfigUtils which exposes the existing misconfigurations in this test. Since these test are checking for misconfiguration, adding a standard test util method to fetch streamconfigs is going to help much. So, I think this change can remain.

Reply:
We will eventually remove the notion of consumerType, anyway